packages/cli/src/runner/crawler-command-runner.ts

```diff
@@ -42,35 +44,46 @@ export class CrawlerCommandRunner implements CommandRunner {
 
         const endDate = new Date();
         const endDateNumber = Date.now();
+
+        await this.generateSummaryReports(scanArguments, startDate, startDateNumber, endDate, endDateNumber);
+    }
+
+    private async generateSummaryReports(
+        scanArguments: ScanArguments,
+        startDate: Date,
+        startDateNumber: number,
+        endDate: Date,
+        endDateNumber: number,
+    ): Promise<void> {
         const durationSeconds = (endDateNumber - startDateNumber) / 1000;
         console.log(`Done in ${durationSeconds}s`);
 
-        const scannedPagesCount = scanResult.summaryScanResults.failed.length + scanResult.summaryScanResults.passed.length;
-        const discoveredPagesCount = scannedPagesCount + scanResult.summaryScanResults.unscannable.length;
+        const scannedPagesCount = this.scanResult.summaryScanResults.failed.length + this.scanResult.summaryScanResults.passed.length;
```

Comment:
Should we be null checking the scan result? As in, are we always sure it will have these properties and what not?

Reply:
we don't need to check at least for now, if the crawler package has no results, it will return an object empty with arrays of length 0 eg:
scanResult : {
summaryScanResults: {
failed:[],
passed:[],
unscannable:[],
}
}